torch/csrc/inductor/aoti_torch/tensor_converter.h

```diff
@@ -0,0 +1,34 @@
+#pragma once
+
+#include <torch/csrc/inductor/aoti_torch/c/shim.h>
+
+#include <ATen/Tensor.h>
+
+namespace torch {
+namespace aot_inductor {
+
+// Functions declared here are not meant to be called from the AOTInductor
+// generated model.so
+
+TORCH_API at::Tensor* tensor_handle_to_tensor_pointer(AtenTensorHandle handle);
+
+TORCH_API AtenTensorHandle tensor_pointer_to_tensor_handle(at::Tensor* tensor);
+
+// borrow_tensors_to_handles is used for turning a vector of aten tensors into
+// a vector of AtenTensorHandles, and then pass that into model.so.
+//
+// Let's start with borrowing ownership instead of stealing, correctness first
+// and then optimize for performance
```

Comment:
This is not... really a thing? Like a borrow costs the same as a steal; it's the new reference that is expensive